subjects/graphql/README.md

```diff
@@ -0,0 +1,185 @@
+## GraphQL
+
+### Objectives
+
+The objective of this project is to learn graphql query language by creating profiles. It will be provided,\
+by the platform, a graphql endpoint that is connected to the database. So you can query this endpoint to obtain the information you desire.\
+**Note** that for security reasons some tables are private and some are public, you will only be provided with certain content.
+
+Your profile must have at least:
+
+- Basic user identification
+- XP amount
+- level
+- grades
+- skills
+- statistics
+
+### Instructions
+
+> Here is a small introduction and advantages of graphQL, if you want to read more about graphQL you can visit there [site](https://graphql.org/)
+
+GraphQL is a query language for any API endpoint and runtime. The syntax lets you specify what data you want to receive from that API.\
```

Comment:
I don't think we have to explain what graphQL is, they can look it up.

Reply:
ok, on it